vuepress/docs/next/docs/README.md

```diff
@@ -12,28 +12,56 @@ only be available by building from source.
 
 Entando is the leading open source **Application Composition Platform** for Kubernetes. It enables parallel teams to accelerate development and innovation of business capabilities on a modern, cloud-native, and Kubernetes-native stack.
 
-With native support for Javascript app development, a micro frontend and microservices architecture, deployment of apps in containers that can be individually scaled up and down, and automated management of containers with Kubernetes, we simplify the move for enterprises looking to modernize across on-prem and cloud infrastructures.
+It is a distributed and composable environment in which components can be built, scaled and iterated independently, helping teams build agile applications that can adapt to change.
 
-## Entando Architecture
+![Entando Platform Development Process](./getting-started/img/concepts.png) 
 
-![entando-architecture](./entando-architecture.png)
+On the Entando Platform, development of an application is organized into four stages: Create, Curate, Compose, and Consume. In the simplest terms, you create functional building blocks, curate them into a catalog, compose the blocks onto an app or website, which is then consumed by a user.
```

Comment:
the second sentence doesn't make sense as-is. simplest option is to add "and" before "compose the blocks." without "and" it's grammatically incorrect.